lib/enhancements/edition.rb

```diff
@@ -140,4 +140,13 @@ def artefact
   def self.convertible_formats
     Artefact::FORMATS_BY_DEFAULT_OWNING_APP["publisher"] - ["local_transaction"]
   end
+
+  def fact_check_id
```

Comment:
@danielroseman Rather than introduce the complexity of managing a (randomly generated) UUID over the object lifecycle, we opted for a more functional approach. The `#fact_check_id` method delivers an encoding of the `Edition.id` that validates as UUID only when the content workflow should allow fact-check access.